lib/child_process.js

```diff
@@ -378,14 +378,13 @@ function execFile(file /* , args, options, callback */) {
     if (options.signal.aborted) {
       process.nextTick(() => kill());
     } else {
+      const childController = new AbortController();
       options.signal.addEventListener('abort', () => {
-        if (!ex) {
```

Comment:
Not blocking but: Unrelated style change?

Reply:
Yeah I just noticed it's different from the "common" way in other places in the event target code so I fixed my code there. Putting it in a new/different PR/commit seemed like overkill I think. I don't mind reverting it if it's an issue.

Reply:
Nah, I just wanted to make sure there was a reason beyond personal preference. The inconsistency in curly brace usage throughout the code base bugs me, and I want to make sure any changes have a justification beyond personal preference. It will at least allow us to inch towards consistency. Maybe we'll get there by about 2027.